src/libstd/sys/unix/process/process_common.rs

```diff
@@ -149,11 +149,19 @@ impl Command {
         }
     }
 
+    pub fn set_arg_0(&mut self, arg: &OsStr) {
+        // Set a new arg0
+        let arg = os2c(arg, &mut self.saw_nul);
+        debug_assert!(self.argv.0.len() > 1);
```

Comment:
`debug_assert!(!self.argv.0.is_empty());` ?

Reply:
The current code tests for at least two elements.

Reply:
You're right, I'm sorry